src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -2312,3 +2312,102 @@ plugin could be rewritten like this:
       return [MuteController]
 
 
+Style Guide for UI
+------------------
+The style guide is created to document ceph dashbaord standards and maintain
+consistency throught the project. Its an effort to make it easier for
+contributors to process designing and deciding mockups and designs for
+Dashboard.
+
+The development environment for ceph dashbaord has live reloading enabled so
+any changes made in UI are reflected in open browser windows. Ceph dashboard
+uses bootstrap as the main third-party CSS library.
+
+Avoid duplication of code. Be consistent with the existing UI by reusing
+existing scss declarations as much as possible.
+
+Colors
+~~~~~~
+All the colors used in ceph dashboard UI are listed in `frontend/src/styles`.
```

Comment:
The colors are listed in the [_bootstrap-defaults.scss](https://github.com/ceph/ceph/blob/0cf856b04303c9bbf46e334ad85944e543f395a0/src/pybind/mgr/dashboard/frontend/src/styles/defaults/_bootstrap-defaults.scss)
The main color is not `$color-primary` it is `$primary`. Same goes for the `$secondary`

Please keep in mind, that the scss files have been updated recently.